src/ck-core/cklocation.C

```diff
@@ -2507,6 +2533,7 @@ void CkLocMgr::pup(PUP::er& p)
   p | lbmgrID;
   p | metalbID;
   p | bounds;
+  p | idCounter;
```

Comment:
This isn't used elsewhere in this PR, I assume it's a fix for checkpoint/restart?

Reply:
It's related to checkpoint restart, but it is part of the PR. It was previously updated (weirdly) in the old inform method a few lines down. That method was removed with the API updates.